src/Host/Broker/Impl/Startup/CommonStartup.cs

```diff
@@ -4,21 +4,21 @@
 using System;
 using System.IO;
 using System.IO.Pipes;
+using System.ServiceProcess;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Hosting.Server.Features;
 using Microsoft.AspNetCore.Server.Kestrel.Https;
+using Microsoft.Common.Core.Logging;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using Microsoft.R.Host.Broker.Logging;
 using Microsoft.R.Host.Broker.Security;
 using Microsoft.R.Host.Protocol;
 using Newtonsoft.Json;
-using System.Diagnostics;
-using System.ServiceProcess;
 
 namespace Microsoft.R.Host.Broker.Startup {
     internal class CommonStartup {
```

Comment:
IMO too many if(IsService). This is worth separate class, maybe some abstract base and then App and Service